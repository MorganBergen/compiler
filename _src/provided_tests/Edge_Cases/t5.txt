TASKS
    2 5
POLY
    F = x + 1;
    H(X) = F(X) F(X);
EXECUTE
    INPUT Z;
    Z = H(2);
    OUTPUT Z;
INPUTS
    0
