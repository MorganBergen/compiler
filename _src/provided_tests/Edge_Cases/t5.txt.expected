9
F: 1
H: 2
